components/layout_2020/flow/mod.rs

```diff
@@ -257,20 +260,26 @@ impl OutsideMarker {
             },
         );
 
+        // Position the marker beyond the inline start of the border box list item. This needs to
+        // take into account the border of the item.
+        let pbm_of_list_item = self.list_item_style.padding_border_margin(containing_block);
```

Comment:
So this is the containing block passed in when laying out the children of the list item. We don't have access to the writing mode of the list item's parent here -- though this will be affected also by an implementation of `marker-side`. I've modified this to only call `border_width()` making it explicit that only the writing mode is important here. 

What do you recommend?